src/instructlab/training/setup_optimizer.py

```diff
@@ -0,0 +1,39 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Standard
+from typing import Any
+
+# Third Party
+from deepspeed.ops.adam import DeepSpeedCPUAdam, FusedAdam
+import torch
+
+# First Party
+from instructlab.training.config import DistributedBackend
+
+
+def setup_optimizer(args: Any, model: torch.nn.Module) -> torch.optim.Optimizer:
```

Comment:
`args` here is actually not a `typing.Any` but rather a `argparse.Namespace` object:
```suggestion
def setup_optimizer(args: argparse.Namespace model: torch.nn.Module) -> torch.optim.Optimizer:
```